src/PseudoTypes/IntMask.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/*
+ * This file is part of phpDocumentor.
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ *
+ *  @link      http://phpdoc.org
+ *
+ */
+
+declare(strict_types=1);
+
+namespace phpDocumentor\Reflection\PseudoTypes;
+
+use phpDocumentor\Reflection\PseudoType;
+use phpDocumentor\Reflection\Type;
+use phpDocumentor\Reflection\Types\Integer;
+
+use function implode;
+
+/**
+ * Value Object representing the `int-mask` type.
+ *
+ * @psalm-immutable
+ */
+final class IntMask implements PseudoType
```

Comment:
```suggestion
final class IntMask extends Integer implements PseudoType
```

As this class is a pseudo type of an integer, users of this library may expect it to be an interger.